app/shared/storage.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
The logging we use on the UI side of things already breaks this assumption (hence #688), but yeah maybe there's a directory change we could do for 'main process shared' and 'ui shared' and 'both'. Maybe app/main and app/renderer for the main distinction. Also, should make sure we're not putting UI deps in 'dependencies' unless the main process is using which just bloats space (as UI all gets webpacked). Maybe one of the few application structures out there with multiple package.json is the way to go in the future